shell/pages/c/_cluster/uiplugins/__tests__/SetupUIPlugins.test.ts

```diff
@@ -1,128 +1,74 @@
-import { mount, createLocalVue } from '@vue/test-utils';
-import Vuex from 'vuex';
-import {
-  UI_PLUGINS_REPO_URL,
-  UI_PLUGINS_PARTNERS_REPO_URL,
-} from '@shell/config/uiplugins';
-import SetupUIPlugins from '@shell/pages/c/_cluster/uiplugins/SetupUIPlugins.vue';
-
-describe('component: SetupUIPlugins', () => {
-  const localVue = createLocalVue();
+import { mount } from '@vue/test-utils';
 
-  localVue.use(Vuex);
-
-  it('should NOT SHOW a checkbox to install official Rancher repo if NOT prime', async() => {
```

Comment:
Can this condition be checked on the other modal that adds helm repos?

Reply:
This condition is being checked in the [AddExtensionRepos.test.ts](https://github.com/rancher/dashboard/blob/c3cbd738b8b1a4142b7eab7442991f83ffb3f0ed/shell/pages/c/_cluster/uiplugins/__tests__/AddExtensionRepos.test.ts#L32)